peanut_gb.h

```diff
@@ -183,7 +192,11 @@
 /* PPU cycles through modes every 456 cycles. */
 #define LCD_LINE_CYCLES     456
 /* Mode 0 starts on cycle 372. */
+#if PEANUT_FULL_GBC_SUPPORT
+#define LCD_MODE_0_CYCLES   0
```

Comment:
Why does LCD Mode 0 start at cycle 0 in GBC mode? This changes the order of LCD modes to 0->2->3 it seems, rather than 2->3->0.
From https://gbdev.io/pandocs/STAT.html the LCD modes are the same for DMG and CGB, so either this should be changed for both modes, or should be left alone.

![image](https://github.com/deltabeard/Peanut-GB/assets/3747104/2d7af586-b7fb-4136-a3dd-9f1f786e7a91)

What could be an issue is that the number of cycles are static, when Mode 3 can have a variable duration in reality. This is a separate issue/bug to fix however.